build_system/docker_build.py

```diff
@@ -0,0 +1,77 @@
+
+import re
+import subprocess
+import sys
+
+from cross_build import BuildSystem
+import constants as c
+import build_utils as utils
+
+class DockerBuildSystem(BuildSystem):
+    def clean(self, config):
+        try:
+            self.exec_host_cmd("docker rm -f -v j2v8.$PLATFORM.$ARCH", config)
+        except subprocess.CalledProcessError:
+            return
+
+    def health_check(self, config):
+        try:
+            # general docker availability check
+            self.exec_host_cmd("docker stats --no-stream", config)
+
+            # NOTE: the additional newlines are important for the regex matching
+            version_str = utils.execute_to_str("docker version") + "\n\n"
```

Comment:
Ok, looks like a Linux execute permission problem, can you please try executing `chmod +x` for all shell files in `./docker/shared`. I think this will fix this issue, I will see how I can make this automated.